tests/ha/single_pod_restart/test_data_pod_restart.py

```diff
@@ -117,8 +117,8 @@ def setup_method(self):
         LOGGER.info("STARTED: Setup Operations")
         self.random_time = int(time.time())
         self.restore_pod = True
-        self.restore_node = False
-        self.restore_ip = False
+        self.restore_node = True
+        self.restore_ip = True
```

Comment:
Default values should be False only right?

Reply:
No we can keep all true and just chnage once in test code to false once pod is restored or else unncessary we need to chnage it twice once to true and then to false.

Reply:
But restore_ip will only be changed in network down tests.
In teardown it will always go to restore_ip condition. Same is the case with restore_node.